lms/djangoapps/discussion_api/serializers.py

```diff
@@ -172,6 +172,7 @@ class ThreadSerializer(_ContentSerializer):
     comment_list_url = serializers.SerializerMethodField("get_comment_list_url")
     endorsed_comment_list_url = serializers.SerializerMethodField("get_endorsed_comment_list_url")
     non_endorsed_comment_list_url = serializers.SerializerMethodField("get_non_endorsed_comment_list_url")
+    has_endorsed = serializers.BooleanField(read_only=True, source="read")
```

Comment:
I really got my wires crossed while I was writing yesterday. Should be `course="endorsed"`